File: src/main/java/com/davidm1a2/afraidofthedark/common/block/gravewood/BlockGravewoodPlanks.java
package com.davidm1a2.afraidofthedark.common.block.gravewood;

import com.davidm1a2.afraidofthedark.common.block.core.AOTDBlock;
import net.minecraft.block.SoundType;
import net.minecraft.block.material.Material;

/**
 * Class representing gravewood planks
 */
public class BlockGravewoodPlanks extends AOTDBlock
{
    /**
     * Constructor for gravewood planks sets the planks properties
     */
    public BlockGravewoodPlanks()
    {
        super("gravewood_planks", Material.WOOD);
        this.setHardness(2.0f);
        this.setSoundType(SoundType.WOOD);
    }
}

File: src/main/java/com/davidm1a2/afraidofthedark/common/block/gravewood/BlockGravewoodPlanks.kt
package com.davidm1a2.afraidofthedark.common.block.gravewood

import com.davidm1a2.afraidofthedark.common.block.core.AOTDBlock
import net.minecraft.block.SoundType
import net.minecraft.block.material.Material

/**
 * Class representing gravewood planks
 *
 * @constructor for gravewood planks sets the planks properties
 */
class BlockGravewoodPlanks : AOTDBlock("gravewood_planks", Material.WOOD) {
    init {
        setHardness(2.0f)
        this.soundType = SoundType.WOOD
    }
}